letter-shop.md

```diff
@@ -0,0 +1,232 @@
+# Letter Shop Tutorial
+
+## What you need before you start:
+
+* a laptop with an internet connection and NodeJS installed
+* basic knowledge of the command line terminal
+* basic knowledge of JavaScript, including Promises and Buffers
+* (optional) the concept of one-way hash functions
+
+## What you'll learn:
+
+* the concept Hash Time Lock Agreements (HTLAs)
+* the Ledger Plugin Interface (LPI)
+* how to build a paid web app using Interledger!
+
+## Step 1:
+
+Save the following JavaScript as `shop.js`:
```

Comment:
What about making a repo that contains all of the example code that you just clone?

Reply:
yeah, while I was writing #2 I ended up adding the code in this repo, I'll add this tutorial's files to this PR.